scripts/review/runExactHeadProbe.ts

```
@@ -0,0 +1,184 @@
+    return result.status ?? 1;
+  } finally {
+    fs.rmSync(tempRoot, { recursive: true, force: true });
+    const finalStatus = git(repoRoot, [
+      'status',
+      '--porcelain=v1',
+      '--untracked-files=all',
+    ]);
+    if (finalStatus !== sourceStatus) {
+      throw new Error(
+        'Reviewer probe changed the source worktree; cleanup or concurrent-write investigation is required'
+      );
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)